docs/SETUP/BACK_END.md

```diff
@@ -87,41 +120,52 @@ Changes to our codebase should always address an [issue](https://github.com/Code
-- Under APIs and Services -> Credentials, create an API key and put that in the CLEAN_GREEN_GOOGLE_KEY variable
-- Under APIs and Services -> Credentials, create a service account.  After you create the service account you will download the service account private key file named like encoded-keyword-ddd-xxx.json.  Copy that to ~/.config/gcloud/application_default_credentials.json.  This path is specified by default in the volumes section of the docker compose file.
-- Go to Cloud storage -> Buckets and create a new bucket.  Name it logically, e.g. cleanandgreenphl-{your_initials}.  It has to be globally unique.  Grant access to at least write to the bucket to your service account.  Put your bucket name in the GOOGLE_CLOUD_BUCKET_NAME variable.  Make sure the tiles file in your bucket is publicly accessible by following Google's instructions online.
+You can run the tile build locally with `docker-compose run vacant-lots-proj` to create a tile file and upload it to your own GCP bucket. First, create your own GCP account using their free trial. You will need to create the following assets in your GCP account and configure them in the environment variables in the `.env` file:
+
+1. Under APIs and Services -> Credentials, create an API key and put that in the CLEAN_GREEN_GOOGLE_KEY variable.
+2. Under APIs and Services -> Credentials, create a service account. After you create the service account you will download the service account private key file named like encoded-keyword-ddd-xxx.json. Copy that to `~/.config/gcloud/application_default_credentials.json`. This path is specified by default in the volumes section of the docker compose file.
+3. Go to Cloud storage -> Buckets and create a new bucket. Name it logically, e.g. cleanandgreenphl-{your_initials}. It has to be globally unique. Grant access to at least write to the bucket to your service account. Put your bucket name in the GOOGLE_CLOUD_BUCKET_NAME variable. Make sure the tiles file in your bucket is publicly accessible by following Google's instructions online.
+
+Your `/data/.env` file should now look like this:
+
+```sh
+POSTGRES_PASSWORD=a-strong-password-here
+VACANT_LOTS_DB=postgresql://postgres:${POSTGRES_PASSWORD}@localhost/vacantlotdb
+CLEAN_GREEN_GOOGLE_KEY=your-api-key-here
+GOOGLE_CLOUD_BUCKET_NAME=your-bucket-name-here
+```
 
 The python script loads the tiles to Google Cloud as `vacant_properties_tiles_staging.pmtiles`. You can check this tileset by changing the config setting on the frontend `useStagingTiles` to `true`. If the tiles look OK, manually change the name in Google Cloud to remove the `_staging` and archive the previous copy.
 
 #### Google Streetview
 
 To update streetview images, after running the full data script run:
 
-```
+```sh
 docker-compose run streetview
```

Comment:
Let's change all instances of `docker-compose` to `docker compose` since the form has been deprecated.